examples/tests/src/journal.test.ts

```diff
@@ -0,0 +1,73 @@
+import * as fs from "fs";
+import * as mupdf from "mupdf";
+import path from "path";
+import { afterEach, beforeEach, describe, expect, it } from "vitest";
+
+const scriptdir = path.resolve(__dirname);
+const filename = path.join(scriptdir, "..", "test.pdf");
+const data = fs.readFileSync(filename);
+
+describe("mupdfjs metadata tests", () => {
```

Comment:
Name it as a test case for journal operations.

Reply:
Sorry, I just copied your metadata file and forgot to adjust the name. I have corrected the name.